code/modules/spells/spell_types/bloodcrawl.dm

```diff
@@ -14,6 +14,10 @@
 	action_background_icon_state = "bg_demon"
 	var/phased = FALSE
 
+/obj/effect/proc_holder/spell/bloodcrawl/on_lose(mob/living/user)
+	if(phased)
+		user.phasein(get_turf(user), TRUE)
```

Comment:
why are you using the bloodcrawl enter proc for this?

Reply:
Because otherwise I'd have to copypaste the `SEND_SIGNAL` line, and some people hate that.